gslab_scons/builders/build_stata.py

```diff
@@ -44,17 +44,18 @@ def build_stata(target, source, env):
     
     # Set up command line arguments
     cl_arg = misc.command_line_args(env)
-
-    executable       = misc.get_stata_executable(env)
+    executable = misc.get_stata_executable(env)
     command_skeleton = misc.get_stata_command(executable)
-
+    
     try:
         command = command_skeleton % (source_file, cl_arg)
         subprocess.check_output(command, 
                                 stderr = subprocess.STDOUT,
                                 shell  = True)
     except subprocess.CalledProcessError:
         message = misc.command_error_msg("Stata", command)
+        if  env['stata_executable'] not in [None, 'None', '']:
```

Comment:
I see. What happens when user enter a wrong stata-executable and then stata fails? Do we want to raise a comment saying that "maybe you should recheck your stata-executable"?